### ISSUE TITLE
Feburary Meeting Updates

### ISSUE THREAD

**Issue by bertwagner** [OhioNorthDatabaseTraining/Admin#11]
Need to update the following locations with next month's meeting information sometime in the next week or so.
- Website
- Meetup
- LinkedIn
- Twitter
- Email list

I wasn't sure exactly how we decided to divide this up.  @erinstellato do you want to do the Website since you have the speaker info and edit access?  If you let me know once you are done, I can update the remaining 4 locations.

**Comment by erinstellato** [comment]
@bertwagner I will create a post on the website for the February meeting.  Did we therefore decide that we do NOT need it on Meetup first, and we don't care if people RVSP??  Sorry, that part is fuzzy to me :)

**Comment by bertwagner** [comment]
Haha I’m fuzzy too @erinstellato. 

We don’t have control of the Meetup account yet (not sure if we’ll have it today or in a week) so I’m guessing this month it’s better to just do the website first and not worry about RSVP. if we get Meetup in time, I can add it everywhere, otherwise we can reevaluate next month.

**Comment by erinstellato** [comment]
@bertwagner I've created the post on the website: https://ondt.org/  Please let me know if you see any issues with it.  I did put the GoToMeeting link directly in the post, but I did create iCal and Google calendar invites, and it's in there.  Let me know if there are other calendar invite links you think I should add.

**Comment by bertwagner** [comment]
Thanks @erinstellato .  I worked with Hasan to get the meetup account and created an event on there as well: https://www.meetup.com/ohio-north-database-training/events/275862156/ . 

I may have screwed up the GoTo meeting link for Ferburary.  I think the correct version is: https://global.gotomeeting.com/join/961360357 .  

Additionally, the iCal link on the blog works for me, but the Google Calendar appears broken. Might just be me, but thought I'd mention.

**Comment by bertwagner** [comment]
@pmpopovichjr do you have access to add me to the LinkedIn group? Also do you happen to have the Twitter credentials?

**Comment by pmpopovichjr** [comment]
@bertwagner Eric had the twitter. Linkedin is Lisa, Hasan, Me, Luis, Eric. It seems someone already got you added :)

**Comment by bertwagner** [comment]
I think we got everything covered for February 🎉:

- [x] Blog post written 
- [x] User group email list notified
- [x] Twitter tweeted
- [x] Meetup page created (https://www.meetup.com/ohio-north-database-training/events/275862156/)
- [x] LinkedIn event created

The only minor issue is the GoTo meeting link in the iCal and Google Calendar links might be inaccurate (https://github.com/OhioNorthDatabaseTraining/Admin/issues/11#issuecomment-763582547).  I updated the blog post with the Meetup link, but I wasn't sure how to update those two calendar files.  I don't think this is a huge deal since I imagine most people will go through Meetup, so I'm going to close the issue.
